test/thermo/RedlichKwongMFTP_Test.cpp

```diff
@@ -0,0 +1,165 @@
+#include "gtest/gtest.h"
+#include "cantera/thermo/RedlichKwongMFTP.h"
+#include "cantera/thermo/ThermoFactory.h"
+
+
+namespace Cantera
+{
+
+class RedlichKwongMFTP_Test : public testing::Test
+{
+public:
+    RedlichKwongMFTP_Test() {
+        test_phase.reset(newPhase("../data/co2_RK_example.cti"));
+    }
+
+    //vary the composition of a co2-h2 mixture:
+    void set_r(const double r) {
+        vector_fp moleFracs(7);
+        moleFracs[0] = r;
+        moleFracs[2] = 1-r;
+        test_phase->setMoleFractions(&moleFracs[0]);
+    }
+
+    std::unique_ptr<ThermoPhase> test_phase;
+};
+
+TEST_F(RedlichKwongMFTP_Test, construct_from_cti)
+{
+    RedlichKwongMFTP* redlich_kwong_phase = dynamic_cast<RedlichKwongMFTP*>(test_phase.get());
+    EXPECT_TRUE(redlich_kwong_phase != NULL);
+}
+
+TEST_F(RedlichKwongMFTP_Test, chem_potentials)
+{
+    test_phase->setState_TP(298.15, 101325.);
+
+    const double expected_result[9] = {
+        -4.573578067074649e+008,
+        -4.573471163377696e+008,
+        -4.573375748803425e+008,
+        -4.573290065058332e+008,
+        -4.573212695326964e+008,
+        -4.573142485189869e+008,
+        -4.573078484551440e+008,
+        -4.573019904340246e+008,
+        -4.572966083775078e+008
+    };
+
+    double xmin = 0.6;
+    double xmax = 0.9;
+    int numSteps = 9;
+    double dx = (xmax-xmin)/(numSteps-1);
+    vector_fp chemPotentials(7);
+    for(int i=0; i < 9; ++i)
+    {
+        set_r(xmin + i*dx);
+        test_phase->getChemPotentials(&chemPotentials[0]);
+        EXPECT_NEAR(expected_result[i], chemPotentials[0], 1.e-6);
+    }
+}
+
+TEST_F(RedlichKwongMFTP_Test, activityCoeffs)
+{
+    test_phase->setState_TP(298., 1.);
+
+    // Test that mu0 + RT log(activityCoeff * MoleFrac) == mu
+    const double RT = GasConstant * 298.;
+    vector_fp mu0(7);
+    vector_fp activityCoeffs(7);
+    vector_fp chemPotentials(7);
+    double xmin = 0.6;
+    double xmax = 0.9;
+    int numSteps = 9;
+    double dx = (xmax-xmin)/(numSteps-1);
+
+    for(int i=0; i < numSteps; ++i)
+    {
+        const double r = xmin + i*dx;
+        set_r(r);
+        test_phase->getChemPotentials(&chemPotentials[0]);
+        test_phase->getActivityCoefficients(&activityCoeffs[0]);
+        test_phase->getStandardChemPotentials(&mu0[0]);
+        EXPECT_NEAR(chemPotentials[0], mu0[0] + RT*std::log(activityCoeffs[0] * r), 1.e-6);
+        EXPECT_NEAR(chemPotentials[2], mu0[2] + RT*std::log(activityCoeffs[2] * (1-r)), 1.e-6);
+    }
+}
+
+TEST_F(RedlichKwongMFTP_Test, standardConcentrations)
+{
+    EXPECT_DOUBLE_EQ(0.040621990447223283, test_phase->standardConcentration(0));
+    EXPECT_DOUBLE_EQ(0.040621990447223283, test_phase->standardConcentration(1));
```

Comment:
It would be more clear to write the comparison value as `test_phase->pressure()/(test_phase->temperature()*GasConstant`.